brambling/static/brambling/sass/modules/_text.sass

```diff
@@ -70,6 +70,10 @@ h1, .h1
 
 //// Buttons
 
+// There was a weird line height thing going on with buttons.
```

Comment:
Can you be more specific in documenting what this `line-height` declaration accomplishes?